assets/airports/cyhz.json

```diff
@@ -0,0 +1,657 @@
+{
+    "radio": {
+        "twr": "Halifax Tower",
+        "app": "Halifax Arrival",
+        "dep": "Halifax Departure"
+    },
+    "icao": "CYHZ",
+    "iata": "YHZ",
+    "magnetic_north": -18,
+    "ctr_radius": 80,
+    "ctr_ceiling": 12000,
+    "initial_alt": 5000,
+    "position": ["N44d52m52.73", "W063d30m33.62", "477.0ft"],
+    "rr_radius_nm": 5.0,
+    "rr_center": ["N44d52m47.00", "W063d30m37.00"],
+    "has_terrain": true,
+    "wind": {
+        "angle": 37,
+        "speed": 9
+    },
+    "arrivalRunway": "05",
+    "departureRunway": "05",
+    "airspace": [
+        {
+            "floor": 0,
+            "ceiling": 12000,
+            "airspace_class": "E",
```

Comment:
@erikquinn Try this: https://www.navcanada.ca/EN/products-and-services/Documents/DAH_Current_EN.pdf